### PR TITLE
fix: make sure 'enabled' and 'sandbox_enabled' are set to use the sandbox

### PR DESCRIPTION
**Pull request by GGabriele**
Since goks only cares about the plugin validation part of the
Kong Lua tree, it's run in a way that's not mouting the default
configuration file. The sandbox mode is triggered via some
config flags, which were missing (and therefore unset).

This adds a patch change to make sure the sandbox is enabled.

### CODE REVIEW

File: patches/lua-tree.patch [hunk 1]
```diff
@@ -183,7 +189,7 @@ index 625bb0c63..956a09ec9 100644
 -    return re_find(value, uuid_regex, "ioj") and true or nil
 +    return re_match(value, uuid_regex) and true or nil
    end,
- 
+
```
Comment (hbagdi) [inline patches/lua-tree.patch:192]: Why are there so many unrelated changes here?

File: lua-tree/share/lua/5.1/kong/tools/sandbox.lua [hunk 2]
```diff
@@ -46,16 +46,10 @@ end
 
 local lazy_conf_methods = {
   enabled = function(self)
-    return kong and
-           kong.configuration and
-           kong.configuration.untrusted_lua and
-           kong.configuration.untrusted_lua ~= 'off'
+    return true
```
Comment (hbagdi) [inline lua-tree/share/lua/5.1/kong/tools/sandbox.lua:49]: Please add a comment in the code here.

File: plugin/validator_test.go [hunk 3]
```diff
@@ -1913,15 +1913,28 @@ func TestValidator_ValidateAllTypedefs(t *testing.T) {
...
+				"lua_code": "test"
+			}`,
+			wantErr:     true,
+			expectedErr: `{"config":{"lua_code":"expected a map"}}`,
+		},
+		{
+			name: "invalid lua code syntax gets rejected",
+			config: `{
+				"lua_code": {"header": "os.execute('echo 10)"}
+			}`,
```
Comment (hbagdi) [inline plugin/validator_test.go:1933]: Please add a test for 'valid lua code but invalid based on sandboxing rules gets rejected'.

**Review by hbagdi — CHANGES_REQUESTED** [review]

File: patches/lua-tree.patch (hunk 1 above)
Comment (GGabriele) [reply patches/lua-tree.patch:192]: These are just whitespaces strip, not sure why the rebuild is adding them. I'll clean this up

Comment (hbagdi) [reply patches/lua-tree.patch:192]: There are also code changes. Look at the line 184/190.

Comment (GGabriele) [reply patches/lua-tree.patch:192]: Those diffs are part of patch itself and are misleading due to the way these are visually rendered in github: for example line [190](https://github.com/Kong/goks/blob/0588232eac5663a6e0770e624ef16c58374de0b4/patches/lua-tree.patch#L190)

File: lua-tree/share/lua/5.1/kong/tools/sandbox.lua (hunk 2 above)
Comment (GGabriele) [reply lua-tree/share/lua/5.1/kong/tools/sandbox.lua:50]: done

File: plugin/validator_test.go (hunk 3 above)
Comment (GGabriele) [reply plugin/validator_test.go:1933]: done

**Review by hbagdi — APPROVED** [review]
